apps/i18n/studio/en_us.json

```diff
@@ -52,6 +56,8 @@
   "catProcedures": "Functions",
   "catText": "Text",
   "catVariables": "Variables",
+  "caveboy": "cave boy",
+  "cavegirl": "cave girl (Jasmine)",
```

Comment:
I could see it being a bit confusing that we sometimes include the name and sometimes not.